docs/source/ESPEasy/AboutUs.rst

```diff
@@ -1,2 +1,138 @@
+Nodo got its first ESP Wi-Fi plugin, using it as a serial to Wi-Fi bridge.
+At that time we were using the stock firmware that came with the module.
+
+Apr 2015 (ESP Connexio)
+-----------------------
+The ESP Connexio project was initiated as the first effort to port the original
+Nodo code to the ESP platform using early version of the ESP8266 Arduino Core.
+The team had to workaround a lot of issues but got it working reasonably.
+
+May 2015 (ESP Easy initiative)
+------------------------------
+ESP Easy was initiated, mainly because the Nodo Connexio concept was a bit
+over complicated. It had the same event list as the original Nodo project.
+Quite powerful but also complicated to end users. So the idea was to have
+something plug and play to hook up sensors to Domoticz. That decision of
+targeting Domoticz is still present in current source code since many plugins
+from the early days have the values setup in such a way that Domoticz can
+import them with little effort.
+
+Sep 2015 (ESP Easy R020)
+------------------------
+As of R017 the Nodo Plugin mechanism was also implemented within ESP Easy.
+Some other inconvenient bugs were fixed, help buttons added and it was decided
+that R020 could be the first production edition.
+
+ESP Easy was launched on Sourceforge as the very first production edition.
+No programming required.
+
+Feb 2016 (ESP Easy R078)
+------------------------
+To make things even more easy, edition R078 was provided as binary images
+with a simple installer. No need for a complicated Arduino IDE setup.
+And it also introduced OTA, so subsequent updates could be done without
+connecting the module to a serial port.
+
+Aug 2016 (ESP Easy R120)
+------------------------
+Lots of fixes and additional features:
+
+* Rules engine
+* Custom dummy device
+* GlobalSync
+* More commands
+* More controllers
+* More devices
+
+Estimated user count was at this time around 7 000.
+
+Nov 2016 (ESP Easy Mega)
+------------------------
+ESP Easy Mega was initiated (only for ESP Modules with at least 1MB flash
+memory). We had to many limitations in ESP Easy, due to the code size that
+exceeded the 512kB modules flash size. Decided to drop support for these
+classic modules and went forward with this version.
+
+Feb 2017 (Change of roles)
+--------------------------
+Psy0rz took over lead development of ESP Easy Mega (2.0.0), he moved the
+source code onto the GitHub page and started to modularize the code. He also
+made ESP Easy support multiple controllers and fallback Wi-Fi. During this time
+nightly releases was introduced and user counts had grown to around 20 000.
+
+Aug 2018 (New core team)
+------------------------
+With both Mvb and Psy0rz having little time to spend on the project two of the
+most eager contributors Grovkillen (mainly wiki documentation and help tools
+at that time) and TD-er (had already implemented the event based Wi-Fi and
+also introduced chunked webpage) stepped in. The aim for the core team is to
```

Comment:
Chunked transfer was not my initial idea.
If I remember correctly, it was @s0170071

Reply:
Yep, that was one of my few bright moments I can tell my kids about when I am old. ;-)

Reply:
@td-er what else did you already implement when you stepped in?